packages/css/src/less/components/atoms.less

```diff
@@ -0,0 +1,11 @@
+@import './button/index';
```

Comment:
Why some files are located in `./less` and some directly in `./{component}`. Shouldn't we be consistent ?